iodata/test/common.py

```diff
@@ -158,3 +167,32 @@ def check_orthonormal(mo_coeffs, ao_overlap, atol=1e-5):
     message = 'Molecular orbitals are not orthonormal!'
     assert_allclose(mo_overlap, np.eye(mo_count),
                     rtol=0., atol=atol, err_msg=message)
+
+
+def load_one_warning(filename: str, fmt: str = None, match: str = "", **kwargs):
```

Comment:
A very minor (and probably picky) comment: I would use `None` as the default value of `match` instead of `""` which is mutable. I checked `pytest.warns` documentation, and that also uses `None` as default.

Reply:
Thanks for catching this. It's fixed.